### PR TITLE
Issue#200 Make header and Footer full width and navbar fixed

### LINKED ISSUE #200: Make the Header and Footer full Width
## Description
As a contributor,
I need have a full width header and footer,
so that I can ensure better user experience.

## Mocks
screen shot 2018-05-31 at 7 26 03 am

### Update [Required]
- [ ] navbar.component.html
- [ ] navbar.component.scss
- [ ] navbar.component.ts

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
2 hours 

@ancyp @Janiceilene Can you please assign me this issue.

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
Made the Header and footer stretch to the full width of the screen.

Fixes #200 

### Type of Change:
- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

Navbar:
screen shot 2018-06-05 at 10 22 19 pm
Footer:
screen shot 2018-06-05 at 10 22 32 pm

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
  

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/app/navbar/navbar.component.html [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
+        </li>
+        <li class="nav-item waves-light pr-4" mdbRippleRadius>
+            <!--Medium-->
+            <a type="button" class="nav-link btn-floating btn-lg btn-fb" target="_blank" href="https://medium.com/@systers_org">
+                <i class="fa fa-medium fa-2x"></i>
+            </a>
+        </li>
+    </ul>
+    <!--End of Contact Us Social Media Icons-->
+</links>
```
Comment (ancyp) [inline src/app/navbar/navbar.component.html]: indentation

**Comment by ancyp** [comment]
Updated the heading to reflect that you made the navbar fixed :)

File: src/app/navbar/navbar.component.html (hunk 1 above)
Comment (BaaniLeen) [reply src/app/navbar/navbar.component.html]: Sure, will fix that :)

**Comment by BaaniLeen** [comment]
Thanks @ancyp :)

**Comment by ancyp** [comment]
Just noticed the page title needs to be changed from MdbAngularFree-> the one that appears on the tab of the browser.
Bundle it into this or future commits.

**Comment by BaaniLeen** [comment]
PR #206 takes care of both full width as well as mobile responsiveness :)

**Pull request closed without merging**